ldclient.go

```diff
@@ -216,21 +216,26 @@ func matchTarget(targets []TargetRule, user User) bool {
 	return false
 }
 
-func (f Feature) Evaluate(user User) (interface{}, bool) {
+func (f Feature) Evaluate(user User) (value interface{}, rulesPassed bool) {
+	value, _, rulesPassed = f.EvaluateExplain(user)
+	return
+}
+
+func (f Feature) EvaluateExplain(user User) (value interface{}, targetMatch bool, rulesPassed bool) {
 
 	if !*f.On {
-		return nil, true
+		return nil, false, true
```

Comment:
Wouldn't it be useful to provide even more information? For example, which rule matched?